addons/payment_mollie/views/payment_views.xml

```diff
@@ -7,7 +7,7 @@
         <field name="arch" type="xml">
             <xpath expr='//group[@name="acquirer"]' position='inside'>
                 <group attrs="{'invisible': [('provider', '!=', 'mollie')]}">
-                    <field name="mollie_api_key" string="API Key" attrs="{'required': [('provider', '=', 'mollie'), ('state', '!=', 'disabled')]}" password="True"/>
+                    <field name="mollie_api_key" password="True" string="API Key" attrs="{'required': [('provider', '=', 'mollie'), ('state', '!=', 'disabled')]}"/>
```

Comment:
Why this change? The `password` attribute is not essential to the comprehension of the view and should therefore stay at the end of the list of attributes.

Reply:
Same thing for other fields.

Reply:
done :)  (both remarks)